libs/generic-view/store/src/lib/external-providers/reducer.ts

```diff
@@ -33,10 +35,24 @@ export const externalProvidersReducer = createReducer(
         state.google.contacts = { ...action.payload }
       }
     })
-    builder.addCase(setAuthData, (state, action) => {
+    builder.addCase(setGoogleAuthData, (state, action) => {
       if (action.payload) {
         state.google.contacts = { ...action.payload }
       }
     })
+    builder.addCase(outlookAuthorize.fulfilled, (state, action) => {
```

Comment:
it's just a question, not a change request: do we need/want to somehow react to a reject event?